src/components/about/About.tsx

```diff
@@ -0,0 +1,30 @@
+import Image, { StaticImageData } from 'next/image';
+
+import styles from './About.module.scss';
+import ArrowDown from '@/assets/images/icons/arrow-down.svg';
+
+interface AboutContent {
+  title: string;
+  imgSrc: StaticImageData;
+  text: string;
+}
+
+const About = ({ content }: { content: AboutContent }) => {
```

Comment:
Can include a contacts / github section in the future. Or should that be in the footer?